fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalTable.java

```diff
@@ -69,6 +69,8 @@ public class ExternalTable implements TableIf, Writable, GsonPostProcessable {
     protected long timestamp;
     @SerializedName(value = "dbName")
     protected String dbName;
+    @SerializedName(value = "aap")
```

Comment:
ExternalTable will not be persist anymore.
You can try it on master branch by:
1. create catalog
2. set a hive table's property
3. restart FE
4. check if that table still has that property

Reply:
Moved the external table auto analyze policy map to external catalog.